aws/ecs-service/iam.tf

```
@@ -0,0 +1,47 @@
+/* 
+ *  IAM role for fargate instances to use when they are launched
+ */
+
+
+data "aws_iam_policy_document" "ecs_tasks_execution_role_policy" {
```

```suggestion
data "aws_iam_policy_document" "ecs_assume_role_policy" {
```

I would probably be more explicit here, and call it the assume role policy - as it's not got any other policy statements.